src/signerkey.js

```diff
@@ -0,0 +1,68 @@
+import xdr from './generated/stellar-xdr_generated';
+import { StrKey, encodeCheck, decodeCheck } from './strkey';
+
+export class SignerKey {
```

Comment:
I guess this doesn't need types as it's not meant for direct usage?

Reply:
True, but done anyway in https://github.com/stellar/js-stellar-base/pull/520/commits/342a98758cee7584217107db02ece7624a9d4f54